src/frontends/tensorflow/tests/telemetry.cpp

```diff
@@ -38,3 +44,73 @@ INSTANTIATE_TEST_SUITE_P(TFTelemetryTest,
                          FrontEndTelemetryTest,
                          ::testing::Values(getTestData()),
                          FrontEndTelemetryTest::getTestCaseName);
+
+TEST(TFTelemetryTest, test_unsupported_tf1_while) {
+    TelemetryFEParam expected_res;
+    expected_res.m_expected_events = {{
+        std::make_tuple("mo", "op_count", "tf_Placeholder", 2),
+        std::make_tuple("mo", "op_count", "tf_Enter", 1),
+        std::make_tuple("mo", "op_count", "tf_Merge", 1),
+        std::make_tuple("mo", "op_count", "tf_Const", 2),
+        std::make_tuple("mo", "op_count", "tf_Less", 1),
+        std::make_tuple("mo", "op_count", "tf_LoopCond", 1),
+        std::make_tuple("mo", "op_count", "tf_Switch", 1),
+        std::make_tuple("mo", "op_count", "tf_Identity", 1),
+        std::make_tuple("mo", "op_count", "tf_Add", 2),
+        std::make_tuple("mo", "op_count", "tf_NextIteration", 1),
+        std::make_tuple("mo", "op_count", "tf_Exit", 1),
+        // expect error cause event due to operation that fails conversion
+        std::make_tuple("mo", "error cause", "tf_NextIteration", 1),
```

Comment:
Do we plan to support NextIteration this Release?
After that we will have to change this test.
Maybe it's better to prepare some model with a custom op, which we definately do not support?

Reply:
Let me think about such model. I agree because `NextIteration` will be supported only by the legacy frontend

Reply:
Replaced with another test that catches unreal operation `Adddd`